_includes/v19.1/misc/external-urls.md

```diff
@@ -15,6 +15,10 @@
 The location parameters often contain special characters that need to be URI-encoded. Use Javascript's [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) function or Go language's [url.QueryEscape](https://golang.org/pkg/net/url/#QueryEscape) function to URI-encode the parameters. Other languages provide similar functions to URI-encode special characters.
 {{site.data.alerts.end}}
 
+{{site.data.alerts.callout_info}}
+If your environment requires an HTTP or HTTPS proxy server for outgoing connections, you can set the standard `http_proxy` and `https_proxy` environment variables when starting CockroachDB.
```

Comment:
`HTTP_PROXY` and `HTTPS_PROXY` should be capitalized.

Reply:
Done.